posthog/session_recordings/models/session_recording_playlist.py

```diff
@@ -21,6 +25,7 @@ class SessionRecordingPlaylist(FileSystemSyncMixin, models.Model):
     pinned = models.BooleanField(default=False)
     deleted = models.BooleanField(default=False)
     filters = models.JSONField(default=dict)
+    type = models.CharField(max_length=10, choices=PlaylistType.choices, null=True, blank=True)
```

Comment:
```suggestion
    type = models.CharField(max_length=50, choices=PlaylistType.choices, null=True, blank=True)
```

i'd give ourselves a little more space... but that's nitpicking